app/views/aid_applications/duplicates/show.html.erb

```diff
@@ -8,27 +8,50 @@
     <div class="form-card__content">
       <p><%= t('.body') %></p>
     </div>
+    <div class="data-table-container">
+      <table class="data-table js-sticky-header">
+        <thead>
+          <tr> <!-- empty header renders top line on table-->
+            <th></th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% AidApplication.matching_submitted_apps(@aid_application).each do |aid_application| %>
```

Comment:
If there are multiple matching AidApplications, I don't think this view will be usable because there won't be a clear demarcation between each application.

Also, this is deign feedback, but it really needs to include the Application Number so that there is some follow-up ability.